ehitchhiking-ui/src/app/shared/components/blacklist/blacklist-users.ts

```diff
@@ -1,4 +1,6 @@
-export const BLACKLISTUSERS: string[] = [
+export const BLACKLIST_DRIVERS: string[] = ['Kyky', 'Someone'];
```

Comment:
pls use usual names, you will show your modal to the customer tomorrow

Reply:
renamed